numpy/lib/histograms.py

```diff
@@ -17,7 +17,7 @@
 _range = range
 
 
-def _hist_bin_sqrt(x):
+def _hist_bin_sqrt(x, _):
```

Comment:
One convention that I like is to include (we [use it at Google](hhttps://github.com/google/styleguide/blob/gh-pages/pyguide.md)) is to include unused arguments by name, but then delete then at the top of the function, e.g.,
```python
def _hist_bin_sqrt(x, range):
    del range  # unused
    return x.ptp() / np.sqrt(x.size)
```

To quote the Google python style guide:

> Other common forms of suppressing this warning include using '`_`' as the identifier for the unused argument, prefixing the argument name with '`unused_`', or assigning them to '`_`'. These forms are allowed but no longer encouraged. The first two break callers that pass arguments by name, while the latter does not enforce that the arguments are actually unused.

Reply:
Did you see this comment?